touchforms/formplayer/api.py

```diff
@@ -293,7 +295,6 @@ def post_data_helper(d, auth, content_type, url, log=False):
 
 
 def formplayer_post_data_helper(d, auth, content_type, url):
-    d['nav_mode'] = 'prompt'
```

Comment:
looks like nav_mode won't be set to prompt for any domains using the experiment. is that intentional?

Reply:
... damnit, thanks Gio